internal/pkg/manifest/env.go

```diff
@@ -41,6 +69,45 @@ type environmentVPCConfig struct {
 	Subnets subnetsConfiguration `yaml:"subnets,omitempty"`
 }
 
+func (cfg *environmentVPCConfig) loadVPCConfig(vpc *config.CustomizeEnv) {
+	if vpc.IsEmpty() {
+		return
+	}
+	if adjusted := vpc.VPCConfig; adjusted != nil {
+		cfg.loadAdjustedVPCConfig(adjusted)
```

Comment:
What is AdjustedVPCConfig?

Reply:
There are two ways of customizing an environment today with `copilot env init`: 
https://github.com/aws/copilot-cli/blob/c94e2b4e83e663a2a322e18e6bf9bd119b9566d6/internal/pkg/config/env.go#L36-L38

You can either "import" a VPC ID, and subnet IDs that already exists. Or you can let copilot create a new VPC and "adjust" its settings like the CIDR ranges, AZs.

Hope this helps!